backend/packages/Upgrade/src/api/controllers/SettingController.ts

```diff
@@ -30,7 +31,7 @@ export class SettingController {
    */
   @Get()
   public getSetting(): Promise<Setting> {
-    return this.settingService.getClientCheck();
+    return this.settingService.getClientCheck(new UpgradeLogger());
```

Comment:
I think we need a generic logger middleware for other controller because we can create the same sessionId from Upgrade portal with details of user doing the changes and some extra logging

Reply:
Need more discussion on this.

Reply:
Why are we not using the logger instance from the request object?